templates/frontend/objects/article_details.tpl

```diff
@@ -212,24 +212,26 @@
 							{translate key="submission.authorBiography"}
 						{/if}
 					</h2>
+					<ul class="authors">
```

Comment:
Have you checked that this doesn't change the visual display of the author biographies? I think it will change the display to a visual bullet-point list, but we don't want there to be any change to the way biographies appear.

If you take a before/after screenshot that will help evaluate the change.

Reply:
I've used the same elements from the authors list on the top. The ul.authors class removes the bullets. See the before:
![image](https://user-images.githubusercontent.com/4992902/212889266-11d42c0c-0fc4-4821-9332-e570fdeefc83.png)

And the after:
![image](https://user-images.githubusercontent.com/4992902/212889389-38c74bad-e199-449f-a241-57e33633796e.png)

Reply:
In your after screenshot, the style of the affiliation data has changed (grey instead of black). But I can't see any change in this PR that would cause that. What am I missing?

Reply:
The affiliation color has changed because ` ul.authors > li > span.affiliation` has a grey color set on the CSS (see the same color on the top list). I can adjust it on CSS adding a rule for `.author_bios .affiliation` to keep it black.

Reply:
I'd say maybe remove the `<span class="affiliation">` tag from the authors in this section. The name and affiliation can be just one HTML element. That way we don't need to change any existing CSS, and the styles stay the same.